jobs/release-microk8s-capi.yaml

```diff
@@ -16,6 +16,9 @@
               credential-id: cdkbot_dockerhub
               username: DOCKERHUB_USR
               password: DOCKERHUB_PSW
+          - text:
+              credential-id: aws_b64encoded_credentials_clusterapi
+              variable: AWS_B64ENCODED_CREDENTIALS
```

Comment:
Is it possible to document here how this credential is produced and used. I had a bad experience recently grepping through the code on how a secret was produced.